### PR TITLE
Update notifications.md

### PR DESCRIPTION
**Pull request by aquiladayc**
Add sample for Log facet variable. 
slack discussion
https://dd.slack.com/archives/C03MS5SNG/p1601424256275600?thread_ts=1601334801.460900&cid=C03MS5SNG

### Preview

https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/monitors/notifications.md [hunk 1]
```diff
@@ -137,6 +137,20 @@ For example, if your tag is `dot.key.test:five` and your monitor is grouped by `
 {{[dot.key.test].name}}
 ```
 
+#### Log facet for variables
```
Comment (ruthnaebeck) [inline content/en/monitors/notifications.md:140]:
```suggestion
#### Log facet variables
```

File: content/en/monitors/notifications.md [hunk 2]
```diff
@@ -137,6 +137,20 @@ For example, if your tag is `dot.key.test:five` and your monitor is grouped by `
...
+
+In Log monitor, you can group by facets. In that case, that facets can be used as a variable.
```
Comment (ruthnaebeck) [inline content/en/monitors/notifications.md:142]:
```suggestion
Log monitors can use facets as variables if the monitor is grouped by the facets.
```

File: content/en/monitors/notifications.md [hunk 3]
```diff
@@ -137,6 +137,20 @@ For example, if your tag is `dot.key.test:five` and your monitor is grouped by `
 {{[dot.key.test].name}}
 ```
 
+#### Log facet for variables
+
+In Log monitor, you can group by facets. In that case, that facets can be used as a variable. 
+For example, if your facet is `@facet` and your monitor is grouped by that, use:
```
Comment (ruthnaebeck) [inline content/en/monitors/notifications.md:143]:
```suggestion
For example, if your log monitor is grouped by the facet `@facet`, the variable is:
```

File: content/en/monitors/notifications.md [hunk 4]
```diff
@@ -137,6 +137,20 @@ For example, if your tag is `dot.key.test:five` and your monitor is grouped by `
...
+
+```text
+{{@facet.name}}
+```
+
+If your facet is with periods, let's say `@facet.with.dot` use:
```
Comment (ruthnaebeck) [inline content/en/monitors/notifications.md:149]:
```suggestion
If your facet has periods, use brackets around the facet, for example:
```

**Review by ruthnaebeck — CHANGES_REQUESTED** [review]

**Comment by aquiladayc** [comment]
is it fine?

**Review by ruthnaebeck — APPROVED** [review]
👍

**Comment by aquiladayc** [comment]
thank you!

**Pull request merged**